libs/platform/src/lib/components/search-field/search-field.component.html

```diff
@@ -40,12 +40,13 @@
 <ng-template #suggestionMenuTemplate>
     <div class="search-field--dropdown fd-popover__body fd-popover__body--no-arrow" [attr.dir]='dir'
         *ngIf="(dropdownValues$ | async | suggestionMatches:inputText)?.length > 0">
-        <nav class="fd-menu" tabindex="-1" role="menu" [attr.id]="menuId" (keydown)="onKeydown($event)">
+        <nav class="fd-menu" tabindex="-1" role="menu" [ngClass]="{ 'fd-menu--compact': compact}" [attr.id]="menuId" (keydown)="onKeydown($event)">
             <ul class="fd-menu__list">
-                <li fdpSearchFieldSuggestion class="search-field--suggestion-item fd-menu__item"
+                <li fdpSearchFieldSuggestion class="search-field--suggestion-item fd-menu__item fd-menu__link"
                     *ngFor="let value of dropdownValues$ | async | suggestionMatches:inputText"
-                    (keydown.enter)="onItemClick(value)" (click)="onItemClick(value)"
-                    [innerHTML]="value | highlight:inputText:true"></li>
+                    (keydown.enter)="onItemClick(value)" (click)="onItemClick(value)">
+                    <span class="fd-menu__title" [innerHTML]="value | highlight:inputText:true"></span>
+                </li>
```

Comment:
Is it intentional that you are not following the Fundamental-styles structure? 
`fd-menu__item` and `fd-menu__link` are not applied on the same element. 

```
<li class="fd-menu__item">
      <a class="fd-menu__link" href="#">
           <span class="fd-menu__title">Option 1</span>
      </a>
</li>
```

Reply:
This was done because of the way the CDK FocusManager works with menu items. When doing keyboard navigation the CDK FocusManager puts the focus on the "host" item, i.e. the element with "fd-menu__item" and cannot invoke focus on child elements. As a result, if the usual Fundamental-styles structure is used the element with "fd-menu__link" never receives focus and then the default "blue" highlight appears instead of the "dotted outline" as per Fiori specs.

I looked at the Fundamental-Styles code, before going this route, and it looked like there two CSS rules `.fd-menu__item` and `.fd-menu__link` are not dependent/nested, and none of the CSS attributes seemed to conflict. However if this is not the case, please let me know.